static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -22,50 +24,50 @@ import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 interface Props extends RouteComponentProps<{}, {}, {}> {}
 
 export default function FeedbackListPage({}: Props) {
-  const location = useLocation();
-  const query = useFeedbackListQueryParams({
-    location,
-    queryReferrer: 'feedback_list_page',
-  });
-
   const organization = useOrganization();
+  const location = useLocation<FeedbackItemLoaderQueryParams>();
 
+  const queryView = useFeedbackListQueryView({
+    queryReferrer: 'feedback_list_page',
+  });
   const feedbackSlug = decodeScalar(location.query.feedbackSlug);
 
   return (
-    <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
-      <FullViewport>
-        <Layout.Header>
-          <Layout.HeaderContent>
-            <Layout.Title>
-              {t('Bug Reports')}
-              <PageHeadingQuestionTooltip
-                title={t(
-                  'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
+    <FeedbackDataContext queryView={queryView} initialDate={new Date()}>
+      <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
+        <FullViewport>
+          <Layout.Header>
+            <Layout.HeaderContent>
+              <Layout.Title>
+                {t('Bug Reports')}
+                <PageHeadingQuestionTooltip
+                  title={t(
+                    'Feedback submitted by users who experienced an error while using your application, including their name, email address, and any additional comments.'
```

Comment:
Maybe something like
```suggestion
                    'Feedback and bug reports submitted by users on your page.'
```
Could reword this since our new feedback doesn't have to be correlated with errors